src/parse/errors.rs

```diff
@@ -413,7 +424,7 @@ impl Error {
         color: ColorWhen,
     ) -> Self
     where
-        O: Into<String>,
+        O: Into<String> + Copy,
```

Comment:
Why do we need the Copy here? It feels unintuitive given that something that can be converted into a string is most likely not going to be copy-able

Reply:
Thanks, I didn't think about that. I removed that bound by improving how causes are formatted.